exec/java-exec/src/main/java/org/apache/drill/exec/physical/impl/aggregate/InternalBatch.java

```diff
@@ -54,6 +56,7 @@ public InternalBatch(RecordBatch incoming, VectorWrapper[] ignoreWrappers, Opera
       this.sv2 = null;
     }
     this.schema = incoming.getSchema();
+    this.incoming = incoming;
```

Comment:
Are we sure we want to do this? It seems the semantics are that this is a, well, "internal batch" a batch used to hold temporary results. By adding the incoming we are binding this batch to the output of another operator. That seems to defeat the purpose of an internal batch.